File: src/main/java/hassan/personnel/managment/models/dto/PositionInsertDto.java
package hassan.personnel.managment.models.dto;

/**
 * Created by Hassan on 11/19/2016.
 */
public class PositionInsertDto {
    private String title;

    private double startPayment;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public double getStartPayment() {
        return startPayment;
    }

    public void setStartPayment(double startPayment) {
        this.startPayment = startPayment;
    }
}

File: src/main/resources/static/app/models/dto/positionInsertDto.js
/**
 * Created by Hassan on 11/19/2016.
 */
var PositionInsertDto = (function () {
    function PositionInsertDto(title, startPayment) {
        this.setup();
        if (title != undefined)
            this.title = title;
        if (startPayment != undefined)
            this.startPayment = startPayment;
    }
    PositionInsertDto.prototype.setup = function () {
        this.title = null;
        this.startPayment = 0;
    };
    return PositionInsertDto;
}());
